src/base_client.py

```
@@ -0,0 +1,102 @@
+import logging
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)